spec-draft.md

```diff
@@ -136,6 +136,106 @@ Example payload for setting "Airplane Mode":
 Data is the upper bits since in the future we may want to support setting
 certain types of Data the device is capable of. For example 3G, 4G, LTE.
 
+Device Orientation
+------------------
+
+The [JSON Wire Protocol](https://code.google.com/p/selenium/wiki/JsonWireProtocol#/session/:sessionId/orientation)
+already supports a very basic concept of
+orientation, involving two orientation options: portrait and landscape.
+Ultimately this will be subsumed under Device Rotation (see below), but for now
+we continue to support these endopints for mobile devices.
+
+* GET /session/:sessionid/orientation
+    * returns ScreenOrientation
+* POST /session/:sessionid/orientation
+    * accepts a ScreenOrientation
+
+### ScreenOrientation
+
+A `ScreenOrientation` can be one of two strings:
+
+* `LANDSCAPE`
```

Comment:
Do we care that [iOS](https://developer.apple.com/library/ios/featuredarticles/ViewControllerPGforiPhoneOS/RespondingtoDeviceOrientationChanges/RespondingtoDeviceOrientationChanges.html) has more than Landscape and Portrait?